kurtosis-devnet/pkg/kurtosis/endpoints.go

```diff
@@ -53,7 +53,7 @@ func WithDepSets(depsets map[string]descriptors.DepSet) ServiceFinderOption {
 func NewServiceFinder(services inspect.ServiceMap, opts ...ServiceFinderOption) *ServiceFinder {
 	f := &ServiceFinder{
 		services:        services,
-		nodeServices:    []string{"cl", "el"},
+		nodeServices:    []string{"cl", "el", "cl-builder", "el-builder", "conductor", "mev"},
```

Comment:
wdyt of having `rbuilder` instead of having el-builder / cl-builder, considering we don't plan to setup a CL on builder's side anytime soon, and devnet env follows the same convention?

Reply:
that part is really kurtosis-specific, and doesn't necessarily limit how we present the information in the final env.